CMakeLists.txt

```diff
@@ -38,12 +33,23 @@ else()
 endif()
 
 # add SDK
-include_directories(${SDK_INSTALL_DIR}/include)
-link_directories(${SDK_INSTALL_DIR}/lib)
+include(ExternalProject)
+ExternalProject_Add(third_party_dronecode_sdk
+    SOURCE_DIR ${CMAKE_CURRENT_BINARY_DIR}/third_party/DronecodeSDK
+    CMAKE_ARGS
+        -DCMAKE_INSTALL_PREFIX=${CMAKE_CURRENT_BINARY_DIR}/build_dronecode_sdk/install
+    GIT_REPOSITORY https://github.com/Dronecode/DronecodeSDK.git
+    GIT_TAG v0.13.0
+    BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/build_dronecode_sdk
+    #INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/build_dronecode_sdk/install"
+    INSTALL_COMMAND ${CMAKE_COMMAND} --build . --target install
```

Comment:
Does that get installed in `${CMAKE_CURRENT_BINARY_DIR}/build_dronecode_sdk/install`? I would guess it goes on the system :confused:. And why is the line above commented? Either it is superfluous (and we should remove it) or it should be used, right?

Reply:
You always spot these things, thanks!

Reply:
It gets installed there locally because we also have:
```
CMAKE_ARGS
        -DCMAKE_INSTALL_PREFIX=${CMAKE_CURRENT_BINARY_DIR}/build_dronecode_sdk/install
```